sklearn/ensemble/tests/test_voting_classifier.py

```diff
@@ -14,11 +16,28 @@
 from sklearn.svm import SVC
 from sklearn.multiclass import OneVsRestClassifier
 
+
 # Load the iris dataset and randomly permute it
 iris = datasets.load_iris()
 X, y = iris.data[:, 1:3], iris.target
 
 
+def test_unfitted():
+    """Raise error if VotingClassifier was not fitted"""
+    eclf = VotingClassifier(estimators=[('lr1', LogisticRegression()),
+                                        ('lr2', LogisticRegression())])
+    assert_raises(NotFittedError, eclf.predict, X)
+    assert_raises(NotFittedError, eclf.transform, X)
+    try:
+        eclf.predict_proba(X)
+    except NotFittedError:
+        pass
+
+    eclf._estimators = []
```

Comment:
Sorry for not being clear, can you also raise an error  when an empty list is passed as estimators during fit time, so that this will never happen?